.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+<!-- please add a icon to the title of this PR (see https://sigs.k8s.io/cluster-api-provider-nested/VERSIONING.md), and delete this line and similar ones -->
```

Comment:
Any chance we can link out to CAPI's copy instead and not have to copy over the `VERSIONING.md` file?
```suggestion
<!-- please add a icon to the title of this PR (see https://sigs.k8s.io/cluster-api/VERSIONING.md), and delete this line and similar ones -->
```

Reply:
OK, @christopherhein , let's keep the link of cluster-api here